src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties

```diff
@@ -1,2 +1,2 @@
-bitbucket.admin.token=Admin token
+bitbucket.admin.token=Personal token
```

Comment:
I think this said 'Admin' token because we require a personal token from an administrator. Changing this text to just 'Personal' token is a bit confusing because we loose the bit that explains it has to be from an admin

Reply:
How about "Administrator's personal token" or "Personal token with admin privileges" 
(btw is it repo or instance admin?)

Is there somewhere else to call out the permission level if it doesn't make sense to make this field more descriptive?

Reply:
Project admin, you can't create "instance admin" through the UI.

Reply:
Sorry when I said instance admin, I meant should the token belong to someone that *is* an instance admin, because they should have read access to all the projects on the instance.

Reply:
Hmm, how about 'Token value` or `Token text` 
At this point we are indicating what the field input is, so maybe even just `Token` is enough?